packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/ErrorView.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { settings } from 'carbon-components';
+import { ReactComponent as DocumentPreviewIcon } from './icons/document_preview.svg';
+
+const ErrorState = () => {
+  const base = `${settings.prefix}--simple-document__error-view`;
```

Comment:
What about calling this `baseClass` instead of simply `base`? I think that this is clearer and also aligns with what we are using in other components.

Reply:
I see this updated in `SimpleDocument` but not here? And thanks for updating in the other file btw.

Reply:
Good catch--fixed!